apps/src/templates/studioHomepages/CoursesStudentEnglish.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import FindLocalClassBanner from './FindLocalClassBanner';
+import {CourseBlocksHoc} from './CourseBlocks';
+import CourseBlocksGradeBands from './CourseBlocksGradeBands';
+import i18n from "@cdo/locale";
+
+/**
+ * This is the main content for the Courses page for a student using English,
+ * though it may also be shown for a signed-out user using English, and is in
+ * fact the default in such a case.
```

Comment:
Good point.  The intention was to show that signed-out English users viewing /courses can see both regular `/courses` (by clicking "Learn") and `/courses?view=teacher` (by clicking "Teach").